docroot/profiles/custom/sitenow/content/block_content/0c0c1f36-3804-48b0-b384-6284eed8c67e.json

```diff
@@ -1,15 +1,15 @@
 {
     "_links": {
         "self": {
-            "href": "https:\/\/default.local.drupal.uiowa.edu\/block\/351?_format=hal_json"
+            "href": "https:\/\/default.local.drupal.uiowa.edu\/block\/1?_format=hal_json"
```

Comment:
My OCD thanks you!

Reply:
I would like to fix the revision numbers too, but I'm not sure if revision IDs are unique per entity type or shared across all entitities.